hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/bucket/BucketCache.java

```diff
@@ -130,6 +131,11 @@ public class BucketCache implements BlockCache, HeapSize {
   private static final String STRONG_REF_KEY = "hbase.bucketcache.offsetlock.usestrongref";
   private static final boolean STRONG_REF_DEFAULT = false;
 
+  /** The cache age of blocks to check if the related file is present on any online regions. */
+  static final String BLOCK_ORPHAN_GRACE_PERIOD = "hbase.bucketcache.block.orphan.evictgraceperiod";
```

Comment:
nit: Since this configuration is in seconds, can we define it directly in the config, as we usually do? Something like `"hbase.bucketcache.block.orphan.evictgraceperiod.seconds"`.